### PR TITLE
acceptOnTimeout, cookieSecure, acceptOnFirstVisit, and more

### PR DESCRIPTION
**Pull request by webaddicto**
1) Added option to agree and close the banner after N milliseconds: this.options.acceptOnTimeout
2) Added option cookieSecure: this.options.cookieSecure
3) Added option to agree the cookie on first time the user visit a page (but do not close the cookiebanner window): this.options.acceptOnFirstVisit
4) Added option to set moreinfo decoration: this.options.moreinfoDecoration
5) Added option to set moreinfo font weight: this.options.moreinfoFontWeight

### CODE REVIEW

**Comment by senko** [comment]
Hey @webaddicto thanks for the PR. Regarding the %LEARNMORE% alias feature, I’d rather not include it because it’s already possible by customizing message (add the link in the directly) and linkmsg (empty).

I’d like to merge the other commits, but before that could you check indentation / line length? In a couple of places looks like (at least in the github ui) the indentation is not aligned with the nearby code, possibly as a result of mixing tab + spaces. Ideally we’d be using spaces everywhere for indentation.

**Comment by webaddicto** [comment]
@senko have aligned the text and removed %LEARNMORE% references.

I made a small update on the last commit 22c8124, basically have added the check to see if linkmsg is empty:

`'<span>' + this.options.message + (this.options.linkmsg ? ' <a>' + this.options.linkmsg + '</a>' : '') + '</span>';`

What do you think?

File: src/cookiebanner.js [hunk 1]
```diff
@@ -483,11 +483,11 @@ THE SOFTWARE.
               });
             }
 		
-	    // Agree and close banner after N milliseconds
-	    if (this.options.acceptOnTimeout) {
-	      // Validate this.options.acceptOnTimeout as numeric
-	      if(!isNaN(parseFloat(this.options.acceptOnTimeout)) && isFinite(this.options.acceptOnTimeout)) {
-	        setTimeout(function() { self.agree_and_close(); }, this.options.acceptOnTimeout);
+           // Agree and close banner after N milliseconds
```
Comment (senko) [inline src/cookiebanner.js]: This entire block still seems to be indented 1 position left to what it should be? (see the closing brace in line 484)

**Comment by webaddicto** [comment]
Ops, didn't noticed that, should be fine now.

**Comment by zytzagoo** [comment]
lgtm

**Pull request merged**